v3/testdata/dNSNoUnderscoresLongValidity.pem

```diff
@@ -0,0 +1,37 @@
+Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number: 3 (0x3)
+        Signature Algorithm: ecdsa-with-SHA256
+        Issuer: 
+        Validity
+            Not Before: Dec 11 00:00:00 2018 GMT
+            Not After : Jan 11 00:00:00 2019 GMT
+        Subject: 
+        Subject Public Key Info:
+            Public Key Algorithm: id-ecPublicKey
+                Public-Key: (256 bit)
+                pub:
+                    04:9d:46:d8:ce:3d:5f:ff:e2:70:6c:30:04:3f:0e:
+                    aa:b0:7c:50:8c:48:89:6e:ee:19:e6:e4:e4:31:3a:
+                    d0:41:e9:2d:b8:de:ef:99:fc:5e:26:fd:ef:b9:e0:
+                    1a:6a:ab:c5:15:8a:b8:b5:2c:7a:ac:7e:b8:70:61:
+                    58:88:2c:52:38
+                ASN1 OID: prime256v1
+                NIST CURVE: P-256
+        X509v3 extensions:
+            X509v3 Subject Alternative Name: 
+                DNS:no.underscore.com
```

Comment:
ditto re: `.test` fixes

Reply:
Done (thank you for your patience on this, I've been traveling for business and I almost missed a flight trying to juggle these similiar-but-different certs so I just punted for the week).